pkg/kn/commands/revision/revision_list_test.go

```diff
@@ -164,7 +163,7 @@ func createMockRevisionWithParams(name, svcName string) *v1alpha1.Revision {
 	revision := &v1alpha1.Revision{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Revision",
-			APIVersion: "knative.dev/v1alpha1",
+			APIVersion: "serving.knative.dev/v1alpha1",
```

Comment:
The change seems to be at odd with the title...

Reply:
Well, what's odd and the test is broken, as it does not broken of because of the wrong coordinates (in fact it the api group *is* [serving.knative.dev/v1alpha1](https://github.com/knative/serving/blob/master/docs/spec/spec.md#revision), it's just not checked in this test, so it's meaningless anyway.

Happy to revert back to the wrong coordinates though.